Renewed/nl_nl.json

```diff
@@ -361,6 +361,88 @@
+	"block.lotr.aspen_trapdoor": "Abelenhouten valluik",
+	"block.lotr.aspen_pressure_plate": "Abelenhouten drukplaat",
+	"block.lotr.aspen_button": "Abelenhouten knop",
+	"block.lotr.stripped_aspen_log": "Gestripte abelenstam",
+	"block.lotr.stripped_aspen_wood": "Gestript abelenhout",
+	"block.lotr.aspen_beam": "Abelenhouten balk",
+	"block.lotr.aspen_sign": "Abelenhouten bord",
+	"block.lotr.lairelosse_log": "Lairelossëstam",
+	"block.lotr.lairelosse_wood": "Lairelossëhout",
+	"block.lotr.lairelosse_planks": "Lairelossëhouten planken",
+	"block.lotr.lairelosse_leaves": "Lairelossëbladeren",
+	"block.lotr.lairelosse_sapling": "Lairelossëkiemplant",
+	"block.lotr.lairelosse_slab": "Lairelossëhouten plaat",
+	"block.lotr.lairelosse_stairs": "Lairelossëhouten trap",
+	"block.lotr.lairelosse_fence": "Lairelossëhouten hek",
+	"block.lotr.lairelosse_fence_gate": "Lairelossëhouten poort",
+	"block.lotr.lairelosse_door": "Lairelossëhouten deur",
+	"block.lotr.lairelosse_trapdoor": "Lairelossëhouten valluik",
+	"block.lotr.lairelosse_pressure_plate": "Lairelossëhouten drukplaat",
+	"block.lotr.lairelosse_button": "Lairelossëhouten knop",
+	"block.lotr.stripped_lairelosse_log": "Gestripte lairelossëstam",
+	"block.lotr.stripped_lairelosse_wood": "Gestript lairelossëhout",
+	"block.lotr.lairelosse_beam": "Lairelossëhouten balk",
+	"block.lotr.lairelosse_sign": "Lairelossëhouten bord",
+	"block.lotr.cedar_log": "Cederstam",
+	"block.lotr.cedar_wood": "Cederhout",
+	"block.lotr.cedar_planks": "Cederhouten planken",
+	"block.lotr.cedar_leaves": "Cederbladeren",
+	"block.lotr.cedar_sapling": "Cederkiemplant",
+	"block.lotr.cedar_slab": "Cederhouten plaat",
+	"block.lotr.cedar_stairs": "Cederhouten trap",
+	"block.lotr.cedar_fence": "Cederhouten hek",
+	"block.lotr.cedar_fence_gate": "Cederhouten poort",
+	"block.lotr.cedar_door": "Cederhouten deur",
+	"block.lotr.cedar_trapdoor": "Cederhouten valluik",
+	"block.lotr.cedar_pressure_plate": "Cederhouten drukplaat",
+	"block.lotr.cedar_button": "Cederhouten knop",
+	"block.lotr.stripped_cedar_log": "Gestripte cederstam",
+	"block.lotr.stripped_cedar_wood": "Gestript cederhout",
+	"block.lotr.cedar_beam": "Cederhouten balk",
+	"block.lotr.cedar_sign": "Cederhouten bord",
+	"block.lotr.blue_mallorn_torch": "Blauwe mallornfakkel",
+	"block.lotr.green_mallorn_torch": "Groene mallornfakkel",
+	"block.lotr.gold_mallorn_torch": "Gouden mallornfakkel",
+	"block.lotr.silver_mallorn_torch": "Zilveren mallornfakkel",
+	"block.lotr.sandstone_dripstone": "Zandsteendruipsteen",
+	"block.lotr.red_sandstone_dripstone": "Rood zandsteendruipsteen",
+	"block.lotr.edhelvir_ore": "Edhelvirerts",
+	"block.lotr.edhelvir_block": "Edhelvirblok",
+	"block.lotr.high_elven_torch": "Hoge-elfenfakkel",
+	"block.lotr.stone_wall": "Stenen muur",
+	"block.lotr.red_sand_gem": "Rood zandjuweel",
+	"block.lotr.yellow_sand_gem": "Geel zandjuweel",
+	"block.lotr.harad_daisy": "Haradmadeliefje",
+	"block.lotr.southbell": "Zuiderlijkklokje",
```

Comment:
`Zuiderlijk klokje` misschien, of `Zuidklokje`.

Reply:
Ik vond het inderdaad ook een lastig woord. `Zuidklokje` vind ik het beste klinken.